SIT.Manager/Models/Aki/AkiServer.cs

```diff
@@ -0,0 +1,20 @@
+﻿using CommunityToolkit.Mvvm.ComponentModel;
+using Newtonsoft.Json;
```

Comment:
Something I just noticed as well is you're using Newtonsoft.Json I think in a lot of other places we use System.Text.Json so maybe we wanna make sure we pick one or the other for the whole thing rather than using both

Reply:
I kinda just default to using newtonsoft as I've always used it but i dunno if we *should* be using it. I'd need your take on which you'd prefer to use

Reply:
A couple of years ago 100% newtonsoft would have been my answer.

I think now they're basically the same in terms of how you use them and I believe System.Text.Json is slightly more performant as well. Plus it's also baked in rather than having to be a nuget package (not that that is an actual positive or negative)

I tend to just prefer System.Text.Json now but don't think there is much in it.

Reply:
Cool 👍 I'll remove the newtonsoft package and see if swapping to the native lib breaks anything